rosa/tests/test_rosa_login.py

```diff
@@ -0,0 +1,56 @@
+import pytest
+from rosa.cli import is_logged_in, NotLoggedInOrWrongEnvError
+
+
+ROSA_ENV = "rosa_env"
+AWS_REGION_STR = "us-east-1"
+
+
+@pytest.fixture
+def mock_build_execute_command(mocker):
+    return mocker.patch("rosa.cli.build_execute_command")
+
+
+@pytest.fixture()
+def allowed_commands():
+    return ["whoami"]
+
+
+def test_is_logged_in_success(mock_build_execute_command, allowed_commands):
+    mock_build_execute_command.return_value = {"out": {"OCM API": ROSA_ENV}, "err": None}
```

Comment:
Please save and re-use `"out"`, `"err"` and `"OCM API"` as well

Reply:
Added parametrized tests instead, since these values needs to be different for each test case. please see https://github.com/RedHatQE/rosa-python-client/pull/247/commits/a89da3ca099c7160259bcae190f84f565e49754d